source/vibe/core/file.d

```diff
@@ -13,8 +13,11 @@ public import vibe.inet.url;
 import vibe.core.drivers.threadedfile; // temporarily needed tp get mkstemps to work
 import vibe.core.driver;
 
-import std.conv;
-import std.c.stdio;
+static if (__VERSION__ > 2066)
```

Comment:
`static if` seems unnecessary.  `core.stdc` has existed since D 2.021 (according to changelog).